apps/condo/domains/billing/schema/AllResidentBillingReceiptsService.test.js

```diff
@@ -2,1133 +2,493 @@
  * Generated by `createservice billing.BillingReceiptsService --type queries`
  */
 
-const { faker } = require('@faker-js/faker')
+const Big = require('big.js')
 
-const { catchErrorFrom } = require('@open-condo/keystone/test.utils')
 const { makeLoggedInAdminClient } = require('@open-condo/keystone/test.utils')
 
+const {
+    CONTEXT_FINISHED_STATUS: ACQUIRING_CONTEXT_FINISHED_STATUS,
+    CONTEXT_IN_PROGRESS_STATUS: ACQUIRING_CONTEXT_IN_PROGRESS_STATUS,
+} = require('@condo/domains/acquiring/constants/context')
 const {
     completeTestPayment,
     createTestAcquiringIntegrationContext,
+    updateTestAcquiringIntegrationContext,
     createTestAcquiringIntegration,
 } = require('@condo/domains/acquiring/utils/testSchema')
-const { makeClientWithPropertyAndBilling, createTestRecipient } = require('@condo/domains/billing/utils/testSchema')
-const { createTestBillingAccount, createTestBillingProperty, createTestBillingIntegrationOrganizationContext, createTestBillingIntegrationAccessRight } = require('@condo/domains/billing/utils/testSchema')
-const { createTestOrganization } = require('@condo/domains/organization/utils/testSchema')
-const { makeClientWithProperty, createTestProperty } = require('@condo/domains/property/utils/testSchema')
-const { registerServiceConsumerByTestClient, updateTestServiceConsumer, registerResidentByTestClient, createTestResident, ServiceConsumer } = require('@condo/domains/resident/utils/testSchema')
-const { addResidentAccess, makeClientWithResidentUser, makeClientWithSupportUser, makeClientWithServiceUser } = require('@condo/domains/user/utils/testSchema')
-
-const { createTestBillingIntegration, createTestBillingReceipt, updateTestBillingReceipt, ResidentBillingReceipt,
-    generateServicesData,
-} = require('../utils/testSchema')
-
+const { createTestRecipient } = require('@condo/domains/billing/utils/testSchema')
+const {
+    createTestBillingAccount,
+    createTestBillingProperty,
+    createTestBillingIntegrationOrganizationContext,
+    updateTestBillingIntegrationOrganizationContext,
+    createTestBillingIntegrationAccessRight,
+} = require('@condo/domains/billing/utils/testSchema')
+const {
+    CONTEXT_FINISHED_STATUS: BILLING_CONTEXT_FINISHED_STATUS,
+    CONTEXT_IN_PROGRESS_STATUS: BILLING_CONTEXT_IN_PROGRESS_STATUS,
+} = require('@condo/domains/miniapp/constants')
+const { makeClientWithProperty, createTestProperty, updateTestProperty } = require('@condo/domains/property/utils/testSchema')
+const { registerServiceConsumerByTestClient, updateTestServiceConsumer, registerResidentByTestClient } = require('@condo/domains/resident/utils/testSchema')
+const { makeClientWithResidentUser, makeClientWithSupportUser, makeClientWithServiceUser } = require('@condo/domains/user/utils/testSchema')
+
+const { createTestBillingIntegration, createTestBillingReceipt, updateTestBillingReceipt, ResidentBillingReceipt } = require('../utils/testSchema')
```

Comment:
Merge this import with duplicate above with `@condo` notation

Reply:
Done